src/frontend/web_application/src/scenes/Contact/components/PublicKeyForm/style.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../../styles/common';
+@import '../../../../styles/vendor/bootstrap_foundation-sites';
+
+.m-public-key-form {
+  padding-left: map-get($co-form__spacing, 'medium');
+  background-color: $co-color__bg__back--higher;
+
+  &__actions {
+    display: flex;
+  }
+
+  &__button-cancel {
+    margin-left:auto;
```

Comment:
we called this "ninja space", a missing or excess space (or tabs)

(this is obviously not important but sometimes very annoying when refactoring using regexp etc.)